src/controllers/like.controller.js

```diff
@@ -0,0 +1,25 @@
+import models from '../database/models/index';
+import commentHelpers from '../helpers/commentHelpers';
+import responseGenerator from '../helpers/responseGenerator';
+
+const { CommentLike, Comments } = models;
+const { liker } = commentHelpers;
+
+const likeController = async (req, res) => {
+  const { commentId: id } = req.params;
```

Comment:
There seems to be no check for the aspiring `liker` being the author of the comment. This means the author of a comment can like their own comment.

Reply:
I think this is cool, there is no need to check for any aspiring liker because we are not  stating any statistics base on the number of users that like specific comment, any user can like any comment